pixleesdk/src/main/java/com/pixlee/pixleesdk/PXLAlbumFilterOptions.java

```diff
@@ -68,6 +77,26 @@ public String toParamString() {
             jsonFilters.putOpt("has_permission", hasPermission);
             jsonFilters.putOpt("has_product", hasProduct);
             jsonFilters.putOpt("in_stock_only", inStockOnly);
+
+            if (inCategories != null && inCategories.size() > 0) {
+                JSONArray categoriesJson = new JSONArray(inCategories);
+                jsonFilters.put("in_categories", categoriesJson);
+            }
+
+            if(filterByUserhandle != null && !filterByUserhandle.isEmpty()){
+                JSONObject userhandleJson = new JSONObject(filterByUserhandle);
+                jsonFilters.put("filter_by_userhandle", userhandleJson);
+            }
+
+            if(computerVision != null && !computerVision.isEmpty()){
```

Comment:
for these, looking to get some distillery.pixlee.com outputs from the examples for sanity (@firetix  and i did it on ios)

Reply:
https://distillery.pixlee.com/api/v2/albums/204643/photos?api_key=196i8ZzIAhKU8dO2kDe&per_page=20&filters={"computer_vision":{"contains":["hat"]}}&sort={"recency":true,"desc":true}&page=1  returns:

```
{
  "account_id": 830,
  "album_id": 204643,
  "page": 1,
  "per_page": 20,
  "total": 1,
  "next": false,
  "data": [
    {
      "id": 122100640,
      "photo_title": "I'm pumped baseball is back... Are you! #capson #baseballisback #dogsofinstagram",
      "latitude": null,
      "longitude": null,
      "tagged_at": 1459875504000,
      "email_address": null,
      "instagram_followers": 37,
      "twitter_followers": 0,
      "avatar_url": "https:\/\/scontent-dfw5-2.cdninstagram.com\/vp\/e40cee4ed43a9e4191071784a90fafcc\/5DB2129A\/t51.2885-19\/s150x150\/12552445_1078737848814240_898471821_a.jpg?_nc_ht=scontent-dfw5-2.cdninstagram.com",
      "user_name": "28_sweety",
      "connected_user_id": 33021284,
      "source": "instagram",
      "content_type": "image",
      "data_file_name": "cab88c86f19dfdc773ef.jpg",
      "medium_url": "https:\/\/scontent.cdninstagram.com\/t51.2885-15\/s320x320\/e35\/12677208_1785417541688450_625927448_n.jpg?ig_cache_key=MTIyMTg0Njc0ODU3OTExNTU2MA%3D%3D.2",
      "big_url": "https:\/\/scontent.cdninstagram.com\/t51.2885-15\/e35\/12677208_1785417541688450_625927448_n.jpg?ig_cache_key=MTIyMTg0Njc0ODU3OTExNTU2MA%3D%3D.2",
      "thumbnail_url": "https:\/\/scontent.cdninstagram.com\/t51.2885-15\/s150x150\/e35\/12677208_1785417541688450_625927448_n.jpg?ig_cache_key=MTIyMTg0Njc0ODU3OTExNTU2MA%3D%3D.2",
      "source_url": "https:\/\/scontent.cdninstagram.com\/t51.2885-15\/e35\/12677208_1785417541688450_625927448_n.jpg?ig_cache_key=MTIyMTg0Njc0ODU3OTExNTU2MA%3D%3D.2",
      "media_id": "1221846748579115560_2902750395",
      "exist_in": 1,
      "collect_term": "#capson",
      "album_photo_id": 328258677,
      "like_count": 0,
      "share_count": 0,
      "action_link": null,
      "action_link_text": null,
      "action_link_title": null,
      "action_link_photo": null,
      "updated_at": 1562611858496,
      "is_starred": false,
      "approved": true,
      "archived": false,
      "is_flagged": false,
      "album_id": 204643,
      "unread_count": 0,
      "collect_method": "comment_hashtag",
      "title": "I'm pumped baseball is back... Are you! #capson #baseballisback #dogsofinstagram",
      "messaged": false,
      "has_permission": false,
      "awaiting_permission": false,
      "social_user_has_liked": false,
      "platform_link": "https:\/\/www.instagram.com\/p\/BD03yTrBjIo",
      "custom_order": null,
      "has_heavy_permission": false,
      "awaiting_heavy_permission": false,
      "permissioned_at": null,
      "heavy_permissioned_at": null,
      "permissioned_by_user_id": null,
      "locality": null,
      "country": null,
      "is_scheduled": false,
      "scheduler": null,
      "engagement_rate": null,
      "notes": null,
      "flag_reasons": {
        
      },
      "marked_as_spam": false,
      "uploader_additional_fields": {
        
      },
      "is_influencer": false,
      "submitter_link": "",
      "alt_text": "image by 28_sweety containing Clothing, Cap, Blue, Baseball cap, Hat",
      "bounding_box_products": [
        
      ],
      "tag_album_ids": [
        
      ],
      "mentions": [
        
      ],
      "native_likes": 36,
      "native_comments": 0,
      "native_retweets": 0,
      "native_favorites": 0,
      "native_shares": 0,
      "native_followers": 37,
      "language": null,
      "deleted_at": null,
      "video_settings": {
        
      },
      "connected_user_social_id": "",
      "subtype": null,
      "narrow_district": null,
      "created_at": 1560366618249,
      "approved_on_date": 1560366618242,
      "tagged_usernames": [
        
      ],
      "products": [
        
      ],
      "pixlee_cdn_photos": {
        "small_url": "https:\/\/static.pixlee.com\/photos\/122100640\/thumb\/cab88c86f19dfdc773ef.jpg",
        "medium_url": "https:\/\/static.pixlee.com\/photos\/122100640\/medium\/cab88c86f19dfdc773ef.jpg",
        "large_url": "https:\/\/static.pixlee.com\/photos\/122100640\/xl\/cab88c86f19dfdc773ef.jpg",
        "original_url": "https:\/\/static.pixlee.com\/photos\/122100640\/original\/cab88c86f19dfdc773ef.jpg",
        "square_medium_url": "https:\/\/static.pixlee.com\/photos\/122100640\/square_medium\/cab88c86f19dfdc773ef.jpg",
        "attributed_medium_url": "https:\/\/static.pixlee.com\/photos\/122100640\/attributed_medium\/cab88c86f19dfdc773ef.jpg"
      }
    }
  ],
  "message": "Successfully returned album photos",
  "status": 200,
  "sortType": "recency"
}
```

which lines up with what cpanel returns:
![Screen Shot 2019-07-17 at 11 44 33 AM](https://user-images.githubusercontent.com/1392948/61390198-51a07680-a888-11e9-9e11-797fca9a0686.png)